### PR TITLE
fix: Issue #70 (fasta file format)

### PR DESCRIPTION
**Pull request by carlosgjs**
Fixes issue #70 . Fasta file was missing sequence headers.

### CODE REVIEW

File: deployment/deploy.sh [hunk 1]
```diff
@@ -29,4 +29,5 @@ az group create -l $LOC -n $GRP
 az acr create --name $ACR --resource-group $GRP --sku basic --admin-enabled true
 az appservice plan create -g $GRP -n $PLAN -l $LOC --is-linux --sku B3
 az webapp create -g $GRP -p $PLAN -n $APP -i $IMG
-az webapp deployment container config  -g $GRP -n $APP --enable-cd true
\ No newline at end of file
+az webapp deployment container config  -g $GRP -n $APP --enable-cd true
+az webapp restart -g $GRP -n $APP
```
Comment (carlosgjs) [inline deployment/deploy.sh:33]: This change is unrelated/opportunistic

**Review by lsetiawan — APPROVED** [review]
LGTM
